firstvoices/backend/search/documents/base_document.py

```diff
@@ -2,8 +2,14 @@
 
 
 class BaseDocument(Document):
+    """Basic fields used for generic hydration utils"""
+
```

Comment:
I logged fw-5517 to convert the other search endpoints to use the same generic hydration, eventually.